creator/Reference/Content/EntityReference/Examples/EntityComponents/minecraftComponent_environment_sensor.md

```diff
@@ -15,11 +15,34 @@ ms.prod: gaming
 |:----------|:----------|:----------|:----------|
 | triggers| *not set*| List| The list of triggers that fire when the environment conditions match the given filter criteria. |
 
+|Name |Default Value  |Type  |Description  |
+|:----------|:----------|:----------|:----------|
+| event| *not set*| string |The event to run when the conditions for this trigger are met.|
+| filters|*not set*|List / Filter|The list of conditions for this trigger to execute.|
```

Comment:
Will adjust on our end to have Type = `Filter` and add a hyperlink to the filter list to keep in sync.